deploy/complete/terraform/schema.yaml

```diff
@@ -0,0 +1,320 @@
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
The permalink to this `schema.yaml` should be listed somewhere as a reference guide on how to build one. ! Veruy nice !